### DISCUSSION TITLE
How to run `prefect server start` as daemon?

### DISCUSSION THREAD

**Discussion by den-pro-soft** [PrefectHQ/prefect#3583] [General] [answered]
Hello. Hope you're doing well.

I wanna run prefect server as daemon in the EC2  instance. Is there any option that runs the prefect server as deamon?
When I run the command `prefect server start`, I can see that it's running, but not sure how to leave it work after disconnected from the server through terminal.

Hope to hearing from you.
Thanks.

**Comment by zanieb** [comment, marked as answer]
Hi!

In my experience, running something in gnu screen is the easiest way to leave it going after your ssh connection is closed. e.g. `screen -dmS "prefect-server" bash -c 'prefect server start'` you can then view your running screens with `screen -ls` and enter your screen to see what's going on with `screen -rd "prefect-server"`

**Reply by den-pro-soft** [reply]
@madkinsz , thanks for your reply.  What if I wanna keep running agent in the Prefect Cloud, how can I do? The bellow code is what I used to run agent on Prefect Cloud. But, it seems to me that the code is not for running an agent forever. How can I figure it out?
```
flow.register(project_name=<project name>)
flow.run_agent(<token>)
```

**Reply by zanieb** [reply]
I am going to mark this as resolved as this next question is a duplicate of your other thread (https://github.com/PrefectHQ/prefect/discussions/3587)

**Reply by den-pro-soft** [reply]
Sure.
